Sources/SwiftSyntaxBuilder/ConvenienceInitializers.swift

```diff
@@ -13,6 +13,26 @@
 @_spi(RawSyntax) import SwiftParser
 @_spi(RawSyntax) import SwiftSyntax
 
+// MARK: - ArrayElementList
+
+extension ArrayElementListSyntax {
+  public init(expressions: [ExprSyntaxProtocol]) {
+    let elements = expressions.enumerated().map { index, expression in
```

Comment:
It's valid for the last element to have a trailing comma in an array literal, so there's no need for `enumerated()` or index-checking here.

Reply:
SwiftSyntaxBuilder doesn’t add a trailing comma for the last element, so I’d prefer to be consistent between the two initializers her and also not add a trailing comma here either. So: I’d prefer to keep it as you wrote it @gibachan.